MSAL/src/native_auth/public/MSALNativeAuthChallengeTypes.h

```diff
@@ -25,13 +25,21 @@
 #ifndef MSALNativeAuthChallengeTypes_h
 #define MSALNativeAuthChallengeTypes_h
 
-/// The set of capabilities that an application wishes to support for Native Auth operations.
-///
-/// Valid options are:
-/// * OOB: The application can support asking a user to supply a verification code that is sent by email.
-/// * Password: The application can support asking a user to supply a password
+#import <Foundation/Foundation.h>
+
+/** 
```

Comment:
As discussed in Teams, this should change back to `///` style comment

Reply:
Yes, updated now